examples/quickstart_gcp/quickstart_gcp/io_managers.py

```diff
@@ -46,10 +46,11 @@ def handle_output(self, context: OutputContext, obj: pd.DataFrame):
 
     def load_input(self, context: InputContext):
         # upstream_output.asset_key is the asset key given to the Out that we're loading for
-        table_name = context.upstream_output.asset_key.to_python_identifier()
+        table_name = context.asset_key.to_python_identifier()
 
         df = pandas_gbq.read_gbq(
-            f"SELECT * FROM `{self._dataset_id}.{table_name}`", credentials=self._credentials
```

Comment:
These changes look irrelevant. Revert? Same with the other ones below.

Reply:
Ah, these weren't intended-- this is a weird effect of black pre-commit hook we should fix. Black resolves config  to the nearest common ancestor config of all formatted files. When all target files are in a single example, black will resolve to the `pyproject.toml` of the example, which doesn't have our line-length modification (found in our root pyproject.toml). So it shortens these lines. When black ran on pre-commit for this PR this is what happened, since all files are in this example.